### PR TITLE
chore: fixed several quickstarts

### PR DESCRIPTION
**Pull request by jpvajda**
This PR fixes some  misc issues quick starts I saw

1. SVG logo was too small
2. browser example had no summary and was formatted oddly
3. trend micro had a name in the DB config with no spacing

### CODE REVIEW

File: packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json [hunk 1]
```diff
@@ -154,9 +154,7 @@
               }
             ]
           },
-          "linkedEntityGuids": [
-            "MjcyNjYwNXxWSVp8REFTSEJPQVJEfDE1MTgyMDI"
-          ]
+          "linkedEntityGuids": ["MjcyNjYwNXxWSVp8REFTSEJPQVJEfDE1MTgyMDI"]
```
Comment (kidk) [inline packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json:157]: I wonder if these will work if they are imported into other accounts. cc @polfliet

**Review by kidk — CHANGES_REQUESTED** [review]
One open question on the linkedEntityGuids. Everything else looks good 👍

File: packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json (hunk 1 above)
Comment (polfliet) [reply packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json:157]: Here is the response from the Dashboard team:

> Dashcow doesn’t process the linkedEntityGuids it receives, it just stores them in the database. So if another customer imports that dashboards and they don’t have access to MjcyNjYwNXxWSVp8REFTSEJPQVJEfDE1MTgyMDI, then they wouldn’t be able to retrieve the entity outline (name, etc) of that linked dashboard as Nataly pointed out, and facet linking won’t work.

So if I understand correctly the user would still see the dashboard and that specific widget, but would not be able to use the advanced features like facet linking (which is ok I think)

Comment (jpvajda) [reply packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json:157]: @polfliet ok oh, I'm not super familiar with how this works but it sounds like this is 🆗

**Review by polfliet — APPROVED** [review]

**Pull request merged**